tests/integ/test_session.py

```diff
@@ -344,3 +344,20 @@ def check(schema_name: str, expected_name: str) -> None:
     check("A", '"A"')
     check('"a b"', '"a b"')
     check('"a""b"', '"a""b"')
+
+
+def test_use_secondary_roles(session):
+    session.use_secondary_roles("all")
+    session.use_secondary_roles("none")
+    session.use_secondary_roles("ALL")
+    session.use_secondary_roles("NONE")
```

Comment:
Is this part only testing that use_secondary_roles does not throw errors with these arguments?

Reply:
yes